core/compute/compute_common.go

```diff
@@ -193,12 +193,17 @@ func setupVolumes(config *container.Config, instance model.Instance, hostConfig
 	if vMounts := instance.VolumesFromDataVolumeMounts; len(vMounts) > 0 {
 		for _, vMount := range vMounts {
 			storagePool := model.StoragePool{}
-			if ok, err := storage.IsVolumeActive(vMount, storagePool, client); !ok && err == nil {
+			// volume active == exists, possibly not attached to this host
+			if ok, err := storage.IsVolumeActive(vMount, storagePool, client); !ok {
```

Comment:
1. Mix cosmetic change with functional change confuses people. 
2. Either way is fine with me. But there is no issue with the original code, and I cannot say it's much better in modified way. I would prefer to leave the original code untouched.